janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -151,6 +151,11 @@
             new ConfigOption<>(ES_CREATE_NS, "use-external-mappings",
             "Whether JanusGraph should make use of an external mapping when registering an index.", ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Boolean> ALLOW_MAPPING_UPDATE =
+            new ConfigOption<>(ES_CREATE_NS, "allow-mapping-update",
+            "Whether JanusGraph should allow a mapping update when registering an index." +
```

Comment:
Add a space at the end of this sentence (after the period), such that when the two sentences are concatenated, they are separated for readability.